explorer/ast/static_scope.h

```diff
@@ -70,10 +77,16 @@ class StaticScope {
 
  private:
   // Equivalent to Resolve, but returns `nullopt` instead of raising an error
-  // if no definition can be found. Still raises a compilation error if more
-  // than one definition is found.
+  // if no declaration can be found. Still raises a compilation error if more
+  // than one declaration is found.
+  // TODO: This diagnosis of multiple declarations is not implemented yet.
```

Comment:
```suggestion
  // TODO: The diagnosis of multiple declarations is not implemented yet.
```

Reply:
The "This" is to connect this TODO more clearly to the claim in the previous sentence.

Hm, but... I think it'd be better if the comment said what we actually do, with a TODO to do the right thing, rather than the comment saying what we meant to do, with a TODO saying that the comment is wrong. I'll make it so.

Reply:
(I assume you're changing this, but since it's not done, I'll just note "This" to me read as "TryResolve"; part of that may be that the behavior isn't described as a diagnosis, so "This [function's] diagnosis" is my implied reading)